qiskit/visualization/latex.py

```diff
@@ -113,7 +98,6 @@ def __init__(self, qubits, clbits, ops, scale, style=None,
         # presence of "box" or "target" determines row spacing
         self.has_box = False
         self.has_target = False
-        self.reverse_bits = reverse_bits
```

Comment:
?

Reply:
Unrelated to what? reverse_bits is no longer passed, so this must be removed.

Reply:
you are right...